### DISCUSSION TITLE
How to edit liquidsoap configuration to change track every X minutes?

### DISCUSSION THREAD

**Discussion by petrechkiv** [AzuraCast/AzuraCast#3865] [Q&A]
**Using Docker installation method**

Yes

**AzuraCast version**

Stable Release v.0.12.1

**Host Operating System**

ubuntu 18.04

**Describe the bug**

Where can I find an example on liquidsoap to change the configuration manually? From backend not working

**Additional context**

**Comment by appnostic-io** [comment]
Hi!

You might have to elaborate on a use case here as it is not a standard operation of liquidsoap to operate in this manner.  Usually, and under normal circumstances, events are triggered based on the media (suich as a song ending/about to end) or some form of trigger.

Based on latest 1.4.4 documentation, even, there is no means to generate an event at a regular interval such as a timer.  

Your best results might be through orchestrating the logic externally in your favourite language and then making use of liquidsoap's telnet or socket interfaces to programatically operate the server.  In this way you can use cron to fire off a command every (x) minutes and request.dynamic (or other commands) to have the server execute such.
